src/backend/joanie/payment/backends/dummy.py

```diff
@@ -142,6 +143,9 @@ def _get_payment_data(
             payment_info["billing_address"] = billing_address
         if credit_card_token:
             payment_info["credit_card_token"] = credit_card_token
+        if installment:
```

Comment:
```suggestion
```

IMO it's not needed. Line 131 you are not testing if its exists and it should exists.

Reply:
You are right ! since the installment is required from now on, we will always have access to it. Thanks again !